PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -153,12 +153,12 @@ internal class PayPalInternalClient(
             .build()
     }
 
-    private fun isAppSwitchEnabled(payPalRequest: PayPalRequest): Boolean {
+    fun isAppSwitchEnabled(payPalRequest: PayPalRequest): Boolean {
```

Comment:
Should these remain public or be internal at least? Not sure if we want to make them public.

Reply:
These methods are from `PayPalInternalClient`, which is now `internal`. I changed them to `public` because I need access to them for UTs. `PayPalInternalClient` and `PayPalClient` need them to define certain flow criteria, any suggestions? UTs are requiring these methods to be public

Reply:
🤔 not sure if @tdchow has a suggestion on this, ideally we'd like to keep them private. If we are going to make them public temporarily we should likely add some docstrings saying it's not covered by semantic versioning so we can update in the future.

Reply:
I believe @richherrera is correct. Even though we removed the `private` modifiers, since this entire class is `internal` these functions will be hidden from merchants. We're also using these functions in `PayPalClient`. So we'll need to keep them as internal.

Reply:
Nice! Thank you both for the explanation.